app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -249,5 +249,13 @@
             android:title="@string/ignore_hardware_media_buttons_title"
             app:singleLineTitle="false"
             app:iconSpaceReserved="false" />
+
+        <SwitchPreferenceCompat
+            android:defaultValue="false"
+            android:key="@string/share_playlist_with_details_key"
+            android:summary="@string/share_playlist_with_details_settings_item"
+            android:title="@string/share_playlist"
+            app:singleLineTitle="false"
+            app:iconSpaceReserved="false" />
```

Comment:
I am not sure if this is the correct place for this setting. @opusforlife2 Where would you put it?